cmd/tk/workflow.go

```diff
@@ -137,6 +137,16 @@ func showCmd() *cobra.Command {
 	}
 	vars := workflowFlags(cmd.Flags())
 	cmd.Run = func(cmd *cobra.Command, args []string) {
+		fi, err := os.Stdout.Stat()
```

Comment:
This is not required, use the global `interactive` variable instead
https://github.com/grafana/tanka/blob/6b24ca43571d064644c111a3f91eaede03da05e1/cmd/tk/main.go#L32